src/app/order/OrderStatus.tsx

```diff
@@ -111,4 +113,29 @@ const OrderStatusMessage: FunctionComponent<OrderStatusMessageProps> = ({
     }
 };
 
+interface MandateTextProps {
+    provider: string;
+    id: string;
+    mandate?: string;
+}
+
+const MandateText = ({ provider, id, mandate = '' }: MandateTextProps) => (
+    <div data-test="order-confirmation-mandate-text-only">
+        <br />
+        <TranslatedString
+            data={ { provider, mandate } }
+            id={ 'order_confirmation.' + id }
+        />
+    </div>
+);
+
+const MandateLink = ({ provider, id, mandate = '' }: MandateTextProps) => (
```

Comment:
Do we ever want `mandate` to be an empty string? This could lead to a link to `''`